### DISCUSSION TITLE
Add default network configuration to docker execution to disable network access?

### DISCUSSION THREAD

**Discussion by lejtemxviw** [super-linter/super-linter#3334] [Q&A]
Would you guys know if it would be possible to add a default network configuration to the docker execution of the linters to disable network access?  I need to make a case for my team to be allowed to use this on our on-prem GitHub runner, and it might be easier if I could make the point that the software this action downloads and executes would be running on our runner, but would be unable to send any information it might get anywhere outside our local network, or even the runner itself...

**Comment by ferrarimarco** [comment]
This is likely out of scope for super-linter, but rather a more general GitHub Actions question.
